.rubocop.yml

```diff
@@ -8,7 +8,7 @@ AllCops:
   DisplayCopNames: true
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/218261371-91cb6287-578c-4892-8e66-7480317278ec.png)

- [ ] I hope these are the future features of this project :smile: Kindly put OOP School Library future features in this section :blush: